lib/webrick/httpresponse.rb

```diff
@@ -268,11 +291,10 @@ def setup_header() # :nodoc:
       elsif %r{^multipart/byteranges} =~ @header['content-type']
         @header.delete('content-length')
       elsif @header['content-length'].nil?
-        if @body.respond_to? :readpartial
-        elsif @body.respond_to? :call
-          make_body_tempfile
-        else
+        if @body.respond_to?(:bytesize)
           @header['content-length'] = (@body ? @body.bytesize : 0).to_s
```

Comment:
`(@body ? @body.bytesize : 0).to_s` looks weird if you already know that `@body` responds to `bytesize`.  With the change in this PR, probably this should be `@body.bytesize.to_s`

Reply:
Okay, fixed.